### DISCUSSION TITLE
How can $fetch upload files and obtain real-time progress?

### DISCUSSION THREAD

**Discussion by bbhxwl** [nuxt/nuxt#28063] [Questions]
How can $fetch upload files and obtain real-time progress?

**Comment by bradley-varol** [comment]
Why have we left axios in favour of something that doesn't have feature parity? 🙄

**Reply by bbhxwl** [reply]
It's indeed a good question. The server and client recommend using different request methods, and there is no upload progress, which is very troublesome.
